src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder.java

```diff
@@ -306,17 +310,22 @@ public boolean perform(AbstractBuild build, Launcher launcher, BuildListener lis
             writeToLog(String.format("Using App '%s'", env.expand(appArtifact)));
             Upload appUpload = adf.uploadApp(project, appArtifact);
             String appArn = appUpload.getArn();
-            String appName = String.format("%s (Jenkins)", appUpload.getName());
-
+            String deviceFarmRunName = null;
```

Comment:
We have runName as build attribute and deviceFarmRunName is the local variable to get the value after expanding the runName. If i directly assign the expanded value to runName variable, it will change the job instance  value permanently